aeppl/transforms.py

```diff
@@ -56,25 +54,32 @@ class TransformedRV(RandomVariable, metaclass=TransformedRVMeta):
 
 
 @singledispatch
-def _default_transformed_rv(
+def _get_default_transformed_rv_op(
```

Comment:
This looks like a purely refactoring change.  Is it necessary for the functionality changes implied by this commit?  If so, you'll need to clarify; otherwise, it would be better in a separate commit, because then we can focus on the functionality itself.

Reply:
The returned object of that method is no longer a node but the `TransformedRV` `Op`. The reason for that is that I have to create the node for non default transforms anyway in the local optimizer, so there was no reason to specialize the dispatch method as much

Reply:
We definitely don't want to start basing our object names so directly on their exact input and return types.  That information is suffiently conveyed by the type information and docstrings; the names really only need to reflect the basic actions/operations, and some ambiguity/inaccuracy is fine as long as it isn't outright contradictory.

Reply:
I am not usually this nitpick,l with the names.

It is just that in this module there are so many things called `transform*` that it becomes hard to keep track. 

In any case, I am fine with reverting this one. I don't like the long name either.

Reply:
On the pro side, it aligns with the `create_transformed_rv_op` name, and those are returning the same type of object now